packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec

```diff
@@ -112,6 +104,15 @@ fi
 %{foreman_plugin_log}
 
 %changelog
+* Wed Jul 24 2024 Nadja Heitmann <nadjah@atix.de> 0.16.1-1
+- Update to 0.16.1-1
+
+* Wed Jul 24 2024 Nadja Heitmann <nadjah@atix.de> - 0.16.1-1
+- Update to 0.16.1
+
+* Wed Jul 24 2024 Nadja Heitmann <nadjah@atix.de> - 0.16.0-1
+- Update to 0.16.0
```

Comment:
I think you can fold the changelog entries into one (and squash the commits), and then we're good to go here.